tests/ServiceProviderTest.php

```diff
@@ -68,6 +71,8 @@ public function test_application_providers_are_registered()
 
     public function test_extra_providers_are_registered()
     {
+        app()->getProvider(ButlerServiceProvider::class)->registerExtraProviders();
```

Comment:
Since "testbench" does not use the butler-service [Application](https://github.com/glesys/butler-service/blob/master/src/Foundation/Application.php).